mmdet3d/models/data_preprocessors/data_preprocessor.py

```diff
@@ -378,10 +381,50 @@ def voxelize(self, points: List[torch.Tensor]) -> Dict[str, torch.Tensor]:
                 coors.append(res_coors)
             voxels = torch.cat(points, dim=0)
             coors = torch.cat(coors, dim=0)
+        elif self.voxel_type == 'cylindrical':
+            voxels, coors = [], []
+            for i, (res, data_sample) in enumerate(zip(points, data_samples)):
+                rho = torch.sqrt(res[:, 0]**2 + res[:, 1]**2)
+                phi = torch.atan2(res[:, 1], res[:, 0]) / math.pi * 180
```

Comment:
Please check whehter it need to convert radian to angle. To my knowledge, we just keep radian here.